src/test/java/core/basesyntax/operations/BalanceOperationTest.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax.operations;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class BalanceOperationTest {
+    private static Operation operation;
+    private static final int OLD_AMOUNT = 10;
+    private static final int AMOUNT = 20;
+
+    @BeforeClass
+    public static void beforeClass() throws Exception {
+        operation = new BalanceOperation();
+    }
+
+    @Test
+    public void calculateAmountTest_Ok() {
+        int expected = OLD_AMOUNT + AMOUNT;
```

Comment:
better to hardcode expected amount isntead of calculating it in the runtime

Reply:
done